Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalTrg_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+import HLTrigger.HLTfilters.htlHighLevel_cfi
+
```

Comment:
@shervin86 you've got a type here:
`htlHighLevel_cfi` instead of `hltHighLevel_cfi`
This results in: 
```
  File "/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-06-06-2300/python/Calibration/EcalAlCaRecoProducers/ALCARECOEcalTrg_cff.py", line 2, in <module>
    import HLTrigger.HLTfilters.htlHighLevel_cfi
ImportError: No module named htlHighLevel_cfi
```